rflx/generator/session.py

```diff
@@ -594,14 +598,14 @@ def _create_abstract_function(
                 )
             procedure_parameters.append(
                 Parameter(
-                    [ID(a.identifier)],
+                    [a.identifier],
                     ID(const.TYPES_BYTES)
```

Comment:
The constants in `const.` are of type ID already, couldn't they be used directly (more instances in other places)?

Reply:
You are right. I have removed the unnecessary `ID` calls for these constants.